src/style.css

```diff
@@ -1,3 +1,78 @@
+html,
 body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+h1 {
+  margin: 38px 0;
+}
+
+.header {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+}
+
+.nav-links {
+  display: flex;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  display: flex;
+  height: 100%;
+  align-items: center;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
The `.is-active` class changes the color to `#00acdc`. Ensure this color contrast is sufficient for accessibility, especially for users with visual impairments.